hack/test-cmd.sh

```diff
@@ -674,6 +676,14 @@ runTests() {
   # Namespaces #
   ##############
 
+  ### Create a new namespace
+  # Pre-condition: only the "default" namespace exists
+  kube::test::get_object_assert 'namespaces' "{{range.items}}{{$id_field}}:{{end}}" 'default:'
+  # Command
+  kubectl create namespace other
```

Comment:
I'd choose a different name, since the next test is expecting that namespace "other" doesn't exist.